.github/workflows/releasenotes-gen.yml

```diff
@@ -0,0 +1,19 @@
+name: "Run release note generation"
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download checkstyle
+        uses: actions/checkout@v3
+
+      - name: Generate release notes
+        env:
+          READ_ONLY_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ./.ci/releasenotes-gen.sh
```

Comment:
should we removed `|` or it is visual separation from previous line ?

Reply:
Testing

Reply:
It works, not sure why I got a syntax error previously https://github.com/checkstyle/checkstyle/actions/runs/3503839360/jobs/5869149314